xdump/__init__.py

```diff
@@ -1,4 +1,4 @@
 # coding: utf-8
 
 
-__version__ = "0.6.0"
+__version__ = "0.6.1"
```

Comment:
I prefer bumping the package version in a separate commit (I'll do it once the new release is ready)